internal/controllers/admins.go

```diff
@@ -172,5 +183,17 @@ func (s Server) GetAllUserActivePlans(ctx echo.Context) error {
 		return ctx.JSON(http.StatusInternalServerError, model.ErrorResponse(err.Error(), http.StatusInternalServerError))
 	}
 	return ctx.JSON(http.StatusOK, model.SuccessResponse(plandata, http.StatusOK))
+}
 
+func (s Server) AddUpdatOperation(ctx echo.Context) error {
+	updateOperationName := ctx.Param("update_operation")
+	if updateOperationName == "" {
+		return ctx.JSON(http.StatusBadRequest, model.ErrorResponse("Invalid Update Operation", http.StatusBadRequest))
+	}
+	var updateOperation = model.UpdateOperation{Name: updateOperationName}
+	err := s.GORMDB.Debug().Create(&updateOperation).Error
+	if err != nil {
+		return ctx.JSON(http.StatusInternalServerError, model.ErrorResponse(err.Error(), http.StatusInternalServerError))
+	}
+	return ctx.JSON(http.StatusOK, model.SuccessResponse("Success", http.StatusOK))
```

Comment:
There's also a shorthand for this case:

```suggestion
	return ctx.Success(ctx, "Success", http.StatusOK)
```